### ISSUE TITLE
Lesions in hair

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9959]
**Preferred term label:**
Lesions in hair

**Synonyms**
Hairs of the scalp, axillary region, trunk, thighs, groin, eyebrows, and eyelashes

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Pthiriasis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001065
- label: Hair shaft lesion
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: 
- def: Applies to an abnormality that is located on hair shafts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9959
- synonyms:
